tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/AllTestsSuite.java

```diff
@@ -23,7 +23,7 @@
 @RunWith(RedDeerSuite.class)
 @SuiteClasses({
 	ArchetypesTest.class,
-	HTML5Parameterized.class,
+	//HTML5Parameterized.class,
```

Comment:
Plan is to remove this, but we should continue with testing.

Reply:
Fixed.